src/Controller/DefaultController.php

```diff
@@ -186,8 +188,27 @@ protected function isAutoFsAvailable()
      */
     protected function clearCache()
     {
-        $realCacheDir = $this->cacheDir;
-        $this->cacheClearer->clear($realCacheDir);
+        try {
+            $commandAndParams = [
+                'command' => 'cache:clear',
+                '--env' => 'prod'
```

Comment:
Don't do that, leave it delete the actual environment.
I know that for dev this is unneede but in some environments, admitedly mostly only in prod, you might have "optimized" installations that do not include some classes etc. and you might have issues when trying to cache clear them.
So I just wouldn't take the risk. The command will know what environment to delete.

Reply:
Done, thanks, seems reasonable.